torchrl/modules/tensordict_module/actors.py

```diff
@@ -1300,6 +1303,46 @@ def get_value_head(self) -> SafeSequential:
         """Returns the value head."""
         return self.module[2]
 
+    @classmethod
+    def from_lmhead_model(cls, base_model):
```

Comment:
seems very custom to our use case. Not sure it's fit to have it in this class.
My worry is that someone would come on the API doc page of this class and see this, which is out of scope for most usages of the class. 
I would either create a subclass or make it a free function. Wdyt?

Reply:
agreed